go.sum

```diff
@@ -108,9 +108,8 @@ github.com/google/go-cmp v0.2.0/go.mod h1:oXzfMopK8JAjlY9xF4vHSVASa0yLyX7SntLO5a
 github.com/google/go-cmp v0.3.0/go.mod h1:8QqcDgzrUqlUb/G2PQTWiueGozuR1884gddMywk6iLU=
 github.com/google/go-cmp v0.3.1/go.mod h1:8QqcDgzrUqlUb/G2PQTWiueGozuR1884gddMywk6iLU=
 github.com/google/go-cmp v0.4.0/go.mod h1:v8dTdLbMG2kIc/vJvl+f65V22dbkXbowE6jgT/gNBxE=
+github.com/google/go-cmp v0.5.0 h1:/QaMHBdZ26BB3SSst0Iwl10Epc+xhTquomWX0oZEB6w=
```

Comment:
this is a change - it seems to have no implications but still I would prefer if we specify v0.5.1 it seems to have some changes to it, so I have no idea why the `vendor` folder doesn't change more :shrug:

Reply:
`go mod tidy` removes it. As you can note the `go-cmp` has been removed from the indirect list by `tidy`. The `vendor` folder doesn't contain indirect packages.

Reply:
v0.5.1 seems like a small diff from v0.5.0... This is weird but, where did v0.5.1 originally come from? It seems like protobuf v1.4.3 specifies go-cmp v0.4.0 ([src](https://github.com/golang/protobuf/blob/v1.4.3/go.mod)) while grpc v1.41.0 specifies go-cmp v0.5.0 ([src](https://github.com/grpc/grpc-go/blob/v1.41.0/go.mod)) :confused: Was there anything else importing go-cmp with v0.5.1?

Reply:
I guess the unique way I can explain it is that we add it manually for any reason. I'm thinking if it isn't an attempt to overwrite the `go-cmp` version with the fixes from `0.5.1`.

Reply:
Looking at go.sum:
```
9d9cea457 (Roger Chapman    2020-10-21 19:42:32 +1100 107) github.com/google/go-cmp v0.2.0/go.mod h1:oXzfMopK8JAjlY9xF4vHSVASa0yLyX7SntLO5aqRK0M=
9d9cea457 (Roger Chapman    2020-10-21 19:42:32 +1100 108) github.com/google/go-cmp v0.3.0/go.mod h1:8QqcDgzrUqlUb/G2PQTWiueGozuR1884gddMywk6iLU=
9d9cea457 (Roger Chapman    2020-10-21 19:42:32 +1100 109) github.com/google/go-cmp v0.3.1/go.mod h1:8QqcDgzrUqlUb/G2PQTWiueGozuR1884gddMywk6iLU=
9d9cea457 (Roger Chapman    2020-10-21 19:42:32 +1100 110) github.com/google/go-cmp v0.4.0/go.mod h1:v8dTdLbMG2kIc/vJvl+f65V22dbkXbowE6jgT/gNBxE=
8e9367331 (Mihail Stoykov   2021-03-29 18:48:29 +0300 111) github.com/google/go-cmp v0.5.0/go.mod h1:v8dTdLbMG2kIc/vJvl+f65V22dbkXbowE6jgT/gNBxE=
83fcc2303 (Ivan Mirić       2020-08-14 10:32:54 +0200 112) github.com/google/go-cmp v0.5.1 h1:JFrFEBb2xKufg6XkJsJr+WbKb4FQlURi5RUcBveYu9k=
83fcc2303 (Ivan Mirić       2020-08-14 10:32:54 +0200 113) github.com/google/go-cmp v0.5.1/go.mod h1:v8dTdLbMG2kIc/vJvl+f65V22dbkXbowE6jgT/gNBxE=
```
The dates are great here :smile: But it appears that the oldest commit, by Ivan, is actually the initial migration to modules. And Mihail's commit in March was update of grpc to v1.36.1. v0.5.1 looks now like a remnant of the migration...
If it doesn't break anything then we're probably "safe" not to fight with `go mod tidy` result and downgrade go-cmp a bit. Alternatively, this PR can be postponed until next dependency update. WDYT?

Reply:
It seems that this went up and down a couple of times including around moving kafka output out as an extension and dropping it. 
Also it seems like it isn't *at all* in the `vendor` directory so that solves the mystery I was having.

I also did try to do it manually and it *worked* for me and no amount of `go mod tidy`  and or `go mod vendor` reverted my manual changes :shrug: